programs/svm-spoke/src/event.rs

```diff
@@ -113,3 +113,8 @@ pub struct ClaimedRelayerRefund {
     pub claim_amount: u64,
     pub refund_address: Pubkey,
 }
+
+#[event]
+pub struct EmergencyDeletedRootBundle {
```

Comment:
EVM implementation has `EmergencyDeleteRootBundle` event, though that would conflict with instruction context struct. So maybe add comment here to later fix EVM event naming.